release/src/Main.hs

```diff
@@ -54,10 +55,26 @@ main = do
           then do
               $logInfo "Make release"
               releaseToBintray upload releaseDir (map (\(a, (_, outp)) -> (a, outp)) files)
+          else $logInfo "Make dry run of release"
+
+      -- Uploading to Maven Central
+      let mavenUploadArtifacts = filter (\a -> getMavenUpload $ artMavenUpload a) artifacts
+      uploadArtifacts <- fmap concat $ forM mavenUploadArtifacts $ \a -> do
+          fs <- artifactCoords optsAllArtifacts a
+          pure fs
+
+      -- Remove permanent upload!!!
+      uploadToMavenCentral (PerformUpload { getPerformUpload = True }) bazelLocations uploadArtifacts
```

Comment:
This comment looks like it should be addressed before we merge this?

Reply:
Absolutely!